### ISSUE TITLE
Virtualbox troubles.

### ISSUE THREAD

**Issue by juanscv** [ozonos/bug-tracker-BETA#9]
Every time i try to open the Activities it crashes and disables all my extensions. 

I am using Virtualbox 4.3.24 on Windows 8.1

3072 MB memory, PIIX3 chipset, I/O APIC, 1 core with PAE/NX, virtualization enabled, Fedora based, 12 video memory.

**Comment by UriHerrera** [comment]
Yes this is a **known issue**. This happens because the shell extension `user-themes` causes the error.

**Comment by juanscv** [comment]
Thanks for the clarification.
